SampleApp/SampleApp.xcodeproj/xcshareddata/xcschemes/SampleApp.xcscheme

```diff
@@ -20,6 +20,62 @@
                ReferencedContainer = "container:SampleApp.xcodeproj">
             </BuildableReference>
          </BuildActionEntry>
+         <BuildActionEntry
```

Comment:
This makes the tutorials build as part of the Cocoapods build; so we can't accidentally break them.